packages/ui-components/src/components/loader/loader.scss

```
@@ -0,0 +1,45 @@
+@import '../../assets/styles/globals';
+
+$pk-loader-overlay-color: #b9c7c9;
```

```suggestion
$loader-overlay-color: var(--loader-overlay-color, kv-color('neutral-4'));
$loader-overlay-opacity: var(--loader-overlay-opacity, 0.4);
$loader-background-color: var(--loader-background-color, kv-color('neutral-1'));
$loader-spinner-color: var(--loader-spinner-color, kv-color('primary'));
```

1- declare the vars at the beginning of the file it's easier to identify all colors used.  
2- with this implementation it's possible to easily change the colors and opacity from outside the lib.
3- we should only use colors in our [color palette](https://kelvininc.github.io/ui-components/?path=/docs/foundation-colors--page).

---

@ricardoarasilva after make the change you need to mark as resolved